webapp/channels/src/components/admin_console/manage_roles_modal/manage_roles_modal.tsx

```diff
@@ -234,6 +234,20 @@ export default class ManageRolesModal extends React.PureComponent<Props, State>
                             />
                         </label>
                     </div>
+                    <p>
+                        <strong>
+                            <FormattedMessage
```

Comment:
Would it make sense to collapse these into a single `FormattedMessage` component? Not sure about the syntax, but something like:
```jsx
<FormattedMessage
    id='admin.manage_roles.additionalRoles_warning_combined'
    defaultMessage='<strong>Note:</strong> <span>The permissions granted above apply to the account as a whole, regardless of whether it is authenticated using a session cookie or a personal access token. For example, selecting post:all will allow the account to post to channels it is not a member of, even without using a personal access token.</span>'
    values={{
        strong: (chunks) => <strong>{chunks}</strong>,
        span: (chunks) => <span className='pt-2 pb-2 light'>{chunks}</span>,
    }}
/>
```

Reply:
yup, makes total sense, thanks for the suggestion